webapidemo/Controllers/BookController.cs

```diff
@@ -16,38 +15,72 @@ public BookController()
 
         // GET: api/Book
         [HttpGet]
-        public List<Book> Get()
+        public ActionResult Get()
         {
-            return bookService.GetBooks();
+            try
+            {
+                return Ok(bookService.GetBooks());
+            }
+            catch (BadRequestException exception)
```

Comment:
No need of Exception here as nothing bad happens here

Reply:
yeah. will do.